extensions/pi-a2a/README.md

```
@@ -94,19 +94,62 @@ Add to `~/.pi/agent/settings.json`:
 }
 ```
 
+### Examples
+
+**Localhost only (default):**
+```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)